app/serializers/rest/status_serializer.rb

```diff
@@ -36,6 +36,17 @@ def current_user?
     !current_user.nil?
   end
 
+  def visibility
+    # This visibility is masked behind "private"
+    # to avoid API changes because there are no
+    # UX differences
```

Comment:
I think it makes sense to add a `limited_audience` array to the API, as a 1) backwards compatible way of indicating the correct visibility and 2) a way for apps to display the list of users who have access to a post.

Reply:
I don't think apps should be able to display a list of users who have access to a post.